cmd/debos/debos.go

```diff
@@ -111,7 +130,7 @@ func main() {
 		ret := m.RunInMachineWithArgs(args)
 
 		if ret != 0 {
-			os.Exit(ret)
+			panic(code1)
```

Comment:
Mh, is `RunInMachineWithArgs()` returning the exit code of the launched command?

Is it worth communicating it back untranslated to the `debos` caller as the previous code did, or it would just be confusing?